src/azure-cli-nspkg/azure/cli/__init__.py

```diff
@@ -0,0 +1,2 @@
+import pkg_resources
+pkg_resources.declare_namespace(__name__)
```

Comment:
This change shouldn't be checked in unless you've talked to @johanste about this...